### ISSUE TITLE
Create high score feature?

### ISSUE THREAD

**Issue by connorphee** [connorphee/color-game#2]
It would be really cool if the game kept track of user's high scores. I think the best way to do this would be modify the game to end when a user guesses incorrectly, and the score is the number guessed correctly before getting one wrong. Maybe when the game ends if the user is in the top x scores, they are prompted for a name to be placed in the newly created high score list.

**Comment by connorphee** [comment]
This would also require turning this into a full stack app with DB behind it. Not a lot of server side work but would have to remove from github pages and host somewhere like Heroku

**Comment by ghost** [comment]
by high score system do you mean "all time" high score across all users or a per user personal best?

**Comment by connorphee** [comment]
Yes I am describing all time high score. This would likely mean disregarding any count of incorrect scores and requiring implementation of some sort of data store (likely mongodb) to keep the top x scores.

**Comment by ghost** [comment]
Great in which case i can implement this some point in october. Probably this weekend.

**Comment by connorphee** [comment]
@ccmiller2018 as a heads up, others have shown interest and I will likely merge the first PR that completes the issue. Thank you for the interest!
